### ISSUE TITLE
Start training

### ISSUE THREAD

**Issue by Time0o** [Time0o/pytorch-colorful-colorization#28]
We could now try starting a couple training runs, I expect this might not work as well as we would like in the beginning.

These are the minimal steps I think are necessary to make this work:

1. Choose a training set. I think a couple hundred or thousand imagenet images depicting the same class should do. I.e. you should:
    * choose a class and download the from the offiical imagenet page (I think you need to create an account there)
    * reserve some images for training
    * modify the configuration file so that training will not run on tiny-imagenet (use `ImageFileOrDirectory` and just dump all images into a directory)
    * before starting  training, run evaluation on a couple images and check if the results look okay to confirm there are no problems

2. Start training and record how long each iteration takes approximately, then adjust the total number of iterations so that training will take a couple of hours. Don't forget to adjust the checkpoint spacing so that maybe a dozen will be saved overall.

3. Train with the following configurations (maybe just one a first):
    * with rebalancing, without model initialization (just start training)
    * with rebalancing, with model initialization (use `--init-proto resources/...v2.prototxt` and `--init-model resources/init_v2.caffemodel`),
    * without rebalancing (remove `class_rebal_lambda` from config), again with and without model initialization

4. _IMPORTANT_ : Figure out how to use the authors caffe code to train their model on the same dataset with exactly the same settings (don't use their PyTorch code and really check that the optimization parameters are the same). Save the resulting models somewhere and record the training set loss curves.
